### DISCUSSION TITLE
Java annotations?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#103] [General]
Does Objeck support something similar to this? To be clear, I'm not asking you to support it. I don't want it. I only wanted to know. Frankly speaking, I hated Java annotations. Too bad annotations infected many languages already including PHP.

**Comment by objeck** [comment]
No, it does not. I do not intend to add annotations unless there is a "must-have" need for them.

**Comment by ghost** [comment]
@objeck We are having problems with spam bots here. The discussion section of many projects were flooded with them, the latest victim is us. I don't know if Github has a way to counter this. The best I can do is reporting these spam bots to Github.

Update: edit to remove the screenshot of the spam bot's comment, Github destroyed it.
